docs/photoniq/fingerprinting/benefits.md

```diff
@@ -0,0 +1,65 @@
+---
+sidebar_position: 2
+title: Features and Benefits
+---
+
+For businesses with customers worldwide, Fingerprint provides a comprehensive solution that empowers privacy-first identification for enhanced personalization and customization in developing business strategies.  
+Some of these features include: 
+
+### Resilient Fingerprinting​
+
+The Fingerprint service generates fingerprints that persist even after visitors take actions to avoid tracking. These actions include:
+
+- Using incognito/private browsing modes
+- Clearing cookies
+- Upgrading browser version
+
+### Browser-Side Agent
+
+Fingerprint uses a browser-side agent. This agent (a browser-side JavaScript library) automatically loads, collects, and transmits information about a visitor's browser to the Fingerprint service. Fingerprint processes this information using proprietary algorithms and advanced machine-learning models to identify web visitors. Some data types collected include user agents, screen resolutions, CPU counts, and others.
```

Comment:
lets remove the word `automatically` for now, as there is some work needed to send the data right now